### PR TITLE
Read bundler version from Gemfile.lock

### PR DESCRIPTION
**Pull request by koral--**
Fixes error when Gemfile.lock is created with bundler v2:
```
You must use Bundler 2 or greater with this lockfile.
Failed to run bundle install, error: exit status 20
```

### CODE REVIEW

**Comment by polok** [comment]
I faced the same issue. I hope your PR will be merged soon!

File: main.go [hunk 1]
```diff
@@ -47,6 +48,17 @@ func failf(format string, v ...interface{}) {
...
 
+func getBundlerVersion() (gems.Version, error) {
+	lockFileContent, err := fileutil.ReadStringFromFile("Gemfile.lock")
+	if err != nil {
+		log.Warnf("Could not read from Gemfile.lock, error: %s", err)
+		log.Infof("Using unspecified bundler version")
+		return gems.Version{}, nil
+	}
+
+	return gems.ParseBundlerVersion(lockFileContent)
```
Comment (trapacska) [inline main.go:59]: is bundler version always specified in gemfile.lock?
not sure if there is an error with `ParseBundlerVersion` then we might better to print the same warning as when failed to read the file itself?

File: main.go [hunk 2]
```diff
@@ -83,25 +95,28 @@ func main() {
 	// Check dependencies
 	log.Infof("Checking dependencies")
 	log.Printf("Bundler...")
-	if ok, err := rubycommand.IsGemInstalled("bundler", ""); err != nil {
+
+	bundlerVersion, err := getBundlerVersion()
```
Comment (trapacska) [inline main.go:99]: lets say here we return gems.Version{}, what will happen with the commands of: IsGemInstalled and InstallBundlerCommand? do they handle empty version?

**Review by trapacska — CHANGES_REQUESTED** [review]

File: main.go (hunk 1 above)
Comment (koral--) [reply main.go:59]: AFAIR Case of missing version is handled here: https://github.com/bitrise-steplib/steps-danger/pull/8/files#diff-7ddfb3e035b42cd70649cc33393fe32cR54

File: main.go (hunk 2 above)
Comment (koral--) [reply main.go:99]: `IsGemInstalled` was used previously with always empty string. So I assume that it is working as intended.
`InstallBundlerCommand` should handle also empty strings: https://github.com/bitrise-io/go-utils/blob/master/command/gems/bundler.go#L14 https://github.com/bitrise-io/go-utils/blob/master/command/gems/gemfile_lock.go#L66

**Review by trapacska — APPROVED** [review]

**Pull request merged**